sanitizers/detectors/template_injection.go

```diff
@@ -1,38 +1,32 @@
 package detectors
 
 import (
-	"errors"
 	"strings"
 	"text/template/parse"
 
 	"github.com/CodeIntelligenceTesting/gofuzz/sanitizers/fuzzer"
+	"github.com/CodeIntelligenceTesting/gofuzz/sanitizers/reporter"
 )
 
 const evilTemplateAction = "{{ .EvilAction }}"
 
-var TemplateInjectionError = errors.New("template injection error")
+var _ Detector = (*TemplateInjection)(nil)
 
 type TemplateInjection struct {
-	id   int         // numeric identifier to distinguish between the detectors for the various call sites
-	tree *parse.Tree // representation of the parsed template
-}
+	id    int         // Numeric identifier to distinguish between the detectors for the various call sites
+	tree  *parse.Tree // Representation of a parsed template
+	vargs []any       // Supplemental arguments that allow a more verbose error reporting
```

Comment:
Is `vargs` needed? It is not used by any of the hooks

Reply:
Actually not anymore, hope that was the last artifact.